docs/ENVIRONMENT.rst

```diff
@@ -142,6 +142,8 @@ REST API
 -  **PATRONI\_RESTAPI\_KEYFILE**: Specifies the file with the secret key in the PEM format.
 -  **PATRONI\_RESTAPI\_CAFILE**: Specifies the file with the CA_BUNDLE with certificates of trusted CAs to use while verifying client certs.
 -  **PATRONI\_RESTAPI\_VERIFY\_CLIENT**: ``none`` (default), ``optional`` or ``required``. When ``none`` REST API will not check client certificates. When ``required`` client certificates are required for all REST API calls. When ``optional`` client certificates are required for all unsafe REST API endpoints. When ``required`` is used, then client authentication succeeds, if the certificate signature verification succeeds. For ``optional`` the client cert will only be checked for ``PUT``, ``POST``, ``PATCH``, and ``DELETE`` requests. 
+-  **PATRONI\_RESTAPI\_HTTP\_EXTRA\_HEADERs**: (optional) HTTP headers let the REST API server pass additional information with an HTTP response.
+-  **PATRONI\_RESTAPI\_HTTPS\_EXTRA\_HEADERs**: (optional) HTTPS headers let the REST API server pass additional information with an HTTP response when TLS is enabled. This will also pass additional information set in http\_extra\_headers.
```

Comment:
`HEADERs` -- the last character is not capitalized.

And one more thing. These parameters supposed to be a YAML object, which translates into the python `dict`. It would be good to mention it here and maybe provide a short example.

Reply:
Please correct me if I am wrong but I feel that newly variables will not be used as environment variables. Similar to 
```
bootstrap:
  dcs:
    postgresql:
      parameters:
```

Reply:
Restapi could be configured only in the patroni.yaml or from environment.
And this [change](https://github.com/zalando/patroni/pull/1645/files#diff-188997f9c97e1dac6c37391a509d13c9R245) actually enables environment variables processing.

Reply:
```
  http_extra_headers:
    'X-Frame-Options': 'SAMEORIGIN'
    'X-XSS-Protection': '1; mode=block'
    'X-Content-Type-Options': 'nosniff'
```
@CyberDem0n what is correct format to translate above in a env variable `export PATRONI_HTTP_EXTRA_HEADERS="..."`  ?

Reply:
@CyberDem0n any guidance ?

Reply:
It could be YAML or JSON
```json
PATRONI_HTTP_EXTRA_HEADERS='{"X-Frame-Options": "SAMEORIGIN", "X-XSS-Protection": "1; mode=block", "X-Content-Type-Options": "nosniff"}'
```

Reply:
Updated